src/components/utils/include/utils/sqlite_wrapper/sql/sql_database.h

```diff
@@ -43,13 +43,13 @@ namespace utils {
 namespace dbms {
 
 class SQLQuery;
-
+enum class Persistent {IN_MEMORY};
```

Comment:
@BSolonenko any reason to make enum with a single value? In this case it can be just replaced with boolean

Reply:
@AKalinich-Luxoft This was taken for better readability of the code.